### PR TITLE
first complete version of the Signal guide

### PR DESCRIPTION
**Pull request by rtanglao**

### CODE REVIEW

**Comment by Twi1ightSparkle** [comment]
Please update the Signal and the WhatsApp guide (and possible other guides where the same is displayed) to use the same image for `1. Click on Signal Bridge in the list of available Bridges`

File: src/Add-Signal-Bridge.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+![](images/sibridge-bot-joins-room.png)  
+1. Open Signal on your mobile device (iOS or Android) and tap on your avatar to go to `Settings` and then `Linked Devices` and then `Link New Device` to start the Signal QR code scanner. You will use this QR code scanner to scan a QR code displayed by your Element client in the next step  
+![](images/sibridge-signal-ios-qrcode.jpg)
+1. From your Element client, send a `link` message to the bot to connect to your Signal account  
+![](images/sibridge-send-link-message.png)
+1. A QR code will be displayed. Quickly scan the QR code with Signal on your mobile device. You have about a minute before it times out. If it times out, just send the login message again to generate another QR code  
+![](images/sibridge-qr-code-from-link-command.png)  
+1. On your Element client, you will see `Successfully logged in as <yourphonenumber>` e.g. `Successfully logged in as +1 555-555-5555` and you will see invitations for each of your Signal chats in your Element client. Each Signal chat is a separate Matrix room. Join one or more chats and start chatting from either your Element app on desktop, iOS or Android or your Signal on mobile.
```
Comment (Twi1ightSparkle) [inline src/Add-Signal-Bridge.md:47]: Is this what signal call them? "A chat"? Not a room, or channel, or something?

Comment (rtanglao) [reply src/Add-Signal-Bridge.md:47]: Yes, Signal calls then `Chats`. It doesn't call them rooms, or channels, i just double checked the Signal on iOS UI.

**Comment by rtanglao** [comment]
> Please update the Signal and the WhatsApp guide (and possible other guides where the same is displayed) to use the same image for `1. Click on Signal Bridge in the list of available Bridges`

is it ok to do this in a separate issue and PR?

**Pull request merged**